src/diamond/metric.py

```diff
@@ -11,7 +11,7 @@ class Metric(object):
     _METRIC_TYPES = ['COUNTER', 'GAUGE']
 
     def __init__(self, path, value, raw_value=None, timestamp=None, precision=0,
-                 host=None, metric_type='COUNTER', ttl=None):
+                 host=None, metric_type='COUNTER', ttl=None, interval=None):
```

Comment:
It would be nice to have an indication of the units of the "interval" argument somewhere.

This doc string seems like the most obvious place to me.